test/functional/tests/cli/test_cli_start_stop.py

```diff
@@ -7,18 +7,15 @@
 import pytest
 from api.cas import casadm, casadm_parser
 from core.test_run import TestRun
-from storage_devices.disk import DiskType
+from storage_devices.disk import DiskType, DiskTypeSet
 from test_utils.size import Unit, Size
 
 
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane]))
```

Comment:
Are cli tests related to the Optanes? I think that all types or at least optane & nand should work here, don't think there's a reason to limit it to Optane only